grails-app/controllers/org/pih/warehouse/api/LoadDataApiController.groovy

```diff
@@ -62,19 +63,31 @@ class LoadDataApiController extends BaseDomainApiController {
         }
 
         if (Boolean.valueOf(config.mainWarehouseInventory.enabled)) {
-            loadDataService.importInventory(new URL(config.mainWarehouseInventory.url))
+            loadDataService.importInventory(
+                    new URL(config.mainWarehouseInventory.url),
+                    Location.findByName('Main Warehouse')
```

Comment:
Let's move the names of warehouses to the config to be alongside URLs.

Reply:
done